errors.go

```diff
@@ -6,7 +6,7 @@ import (
 )
 
 // ErrWrongKind may be returned from functions on the Node interface when
-// a method is invoked which doesn't make sense for the Kind and/or ReprKind
+// a method is invoked which doesn't make sense for the Kind and/or Kind
```

Comment:
I think we can collapse this bit that became "Kind and/or Kind" to just "Kind" now :laughing: 

(I must've written this quite a while ago, when the relationship between the two was less clear.  Now it seems quite clear that only referring to data model Kind here is sufficient.)